actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java

```diff
@@ -854,4 +851,32 @@ private void validateContentOrg(List<String> createdFor) {
       }
     }
   }
+
+  @SuppressWarnings("unchecked")
+  private void syncWithEkstepAndUpdateDB(CourseBatch courseBatch) {
+    if (CourseBatchSchedulerUtil.doOperationInEkStepCourse(
+        courseBatch.getCourseId(), true, courseBatch.getEnrollmentType())) {
+      courseBatch.setCountIncrementStatus(true);
+      try {
+        courseBatch.setCountIncrementDate(ProjectUtil.getFormattedDate());
+        Map<String, Object> courseBatchMap =
+            new ObjectMapper().convertValue(courseBatch, Map.class);
+        Response response = courseBatchDao.update(courseBatchMap);
+
+        if (JsonKey.SUCCESS.equalsIgnoreCase((String) response.get(JsonKey.RESPONSE))) {
+          CourseBatchUtil.syncCourseBatchForeground(
+              (String) courseBatchMap.get(JsonKey.ID), courseBatchMap);
+        } else {
+          ProjectLogger.log(
+              "CourseBatchManagementActor:createCourseBatch: Course batch not synced to ES as response is not successful",
+              LoggerEnum.INFO.name());
+        }
+      } catch (Exception ex) {
```

Comment:
catch multi-exception  in single catch block 
- but do not catch generic exception